opts/validation.go

```diff
@@ -0,0 +1,126 @@
+package opts
+
+import "strings"
+
+var (
```

Comment:
Not sure if it makes sense here but none of those exported errors are commented.

Reply:
Makes no sense :D ?!